reddit_service_websockets/socketserver.py

```diff
@@ -46,6 +45,16 @@ def log_request(self):
         # geventwebsocket logs every request at INFO level. that's annoying.
         pass
 
+    @property
+    def socket_server(self):
+        # this is a pretty nasty smell. we're violating the wsgi model by
+        # expecting the application callable to be of a specific type so that
+        # we can get access to its state for various things (metrics, secrets).
+        # this property at least gets us a little encapsulation.
+        socket_server = self.application.application
```

Comment:
`Sentry` is a WSGI middleware that wraps our application. This grabs the inner application from the outer.

https://github.com/getsentry/raven-python/blob/master/raven/middleware.py#L86-L87